server.js

```diff
@@ -6,6 +6,8 @@ const mesasRoutes = require('./routes/mesas')
 const mesasApiRoutes = require('./routes/api/mesas')
 const ordenesApiRoutes = require('./routes/api/ordenes')
 
+// cambio 
```

Comment:
This could lead to an issue of duplicated IDs.

Proposed change: 

```javascript
export function generateId( ) {
	return Date.now() + '' + Math.random();
}
```